### PR TITLE
781/781a ptsd entry and PtsdType pages

### PR DESCRIPTION
**Pull request by aaronjlech**
## Description
Flattens all PTSD new disabilities, and removes it out of the new disability loop. PTSDs will still come last no matter the order they are entered on the add new disabilities page.
Adds updated intro page for new PTSD disabilities (entry point into 781/781a).
Adds first round of questions for 781/781a -- PTSD TYPE(S).

## Testing done
- [x] Tested locally
- [x] Unit Tests

## Screenshots

**updated PTSD intro page**
image

**Choose PTSD type page**
image

## Acceptance criteria
- [x] PTSD-related disabilities are last, and you only go through the PTSD flow once, regardless of the amount of PTSDs entered.
- [x] Updated PTSD intro page with correct content.
- [x] Created choose PTSD type page(controls the users path into either 781, 781a or both)

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by igreulich — APPROVED** [review]

**Review by devin-mccurdy — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/content/newPTSDFollowUp.jsx [hunk 1]
```diff
@@ -1,17 +1,52 @@
...
+);
+
+export const newPTSDFollowUpDescription = (
+  <div>
     <p>
-      If you need more time to prepare your evidence for your PTSD claim, you
-      can save your application and return to it later when you have your
-      evidence ready to upload.
+      We‘ll now ask you questions about the stressful event or events related to
+      your PTSD. We understand that some of the questions maybe difficult to
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/content/newPTSDFollowUp.jsx]: `may be`?

Comment (aaronjlech) [reply src/applications/disability-benefits/all-claims/content/newPTSDFollowUp.jsx]: @markgreenburg Good catch, fixed.

File: src/applications/disability-benefits/all-claims/pages/choosePtsdType.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { ptsdTypeDescription, ptsdTypeHelp } from '../content/ptsdTypeInfo';
+
+import { disabilityNameTitle } from '../content/newPTSDFollowUp';
+
+export const uiSchema = {
+  'ui:title': disabilityNameTitle,
+  'ui:description': ptsdTypeDescription,
+  'view:selectablePtsdTypes': {
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/pages/choosePtsdType.js]: Should selecting at least one PTSD type be required? If so, here's how we've dealt with that before: https://github.com/department-of-veterans-affairs/vets-website/blob/73c8c90be51c18c35c40590cba9150cdf521b317/src/applications/disability-benefits/526EZ/config/form.js#L309

Comment (markgreenburg) [reply src/applications/disability-benefits/all-claims/pages/choosePtsdType.js]: Actually a better example is https://github.com/department-of-veterans-affairs/vets-website/blob/73c8c90be51c18c35c40590cba9150cdf521b317/src/applications/disability-benefits/all-claims/pages/evidenceTypes.js#L28
There's a `validateBooleanGroup` validator included in forms-system: https://github.com/department-of-veterans-affairs/vets-website/blob/73c8c90be51c18c35c40590cba9150cdf521b317/src/applications/disability-benefits/all-claims/pages/evidenceTypes.js#L1

File: src/applications/disability-benefits/all-claims/tests/pages/choosePtsdType.unit.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+              condition: 'PTSD personal Trauma',
+            },
+          ],
+        }}
+        formData={{}}
+        uiSchema={uiSchema}
+      />,
+    );
+
+    expect(form.find('.usa-input-error-message').length).to.equal(0);
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/tests/pages/choosePtsdType.unit.spec.jsx]: FYI I got annoyed by having to write this out everywhere so I dropped a constant in: https://github.com/department-of-veterans-affairs/vets-website/blob/73c8c90be51c18c35c40590cba9150cdf521b317/src/applications/disability-benefits/all-claims/constants.js#L155

**Review by markgreenburg — APPROVED** [review]
Looks good! I think you might want to validate that a user must check at least one box for the ptsd types, but like I said, lgtm!

**Comment by aaronjlech** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could we please get a code review, and get this merged in when it all looks good at your earliest convenience.

File: src/applications/disability-benefits/all-claims/pages/choosePtsdType.js (hunk 2 above)
Comment (aaronjlech) [reply src/applications/disability-benefits/all-claims/pages/choosePtsdType.js]: We're holding on this validation at the moment, there needs to be a business decision made whether we require them to select at least one or give them the option to select none and fill out 781 as they see fit("everything" is supposed to be optional within this form due to the stressful nature of filling it out).

**Review by robalexdev — APPROVED** [review]

**Pull request merged**